docs: update model-serving with mount vfolder and validation (#65)
lablup/backend.ai-docs-webui: docs/model_serving/model_serving.rst

the service.

.. image:: service-launcher1.png
   :width: 700
   :align: center
   :alt: Service launcher

First, provide a service name. For detailed explanations of each item, please refer to the following:

-  Open To Public: This option allows access to the model service
   without any separate token on the server where the service is to be
   hosted. By default, it is disabled.
-  Model Storage To Mount: This is model folder to mount, which contains 

>    model definition file inside the directory

Reviewer: ```suggestion
   model definition file inside the directory.
```

-  Model Destination For Model Folder: This option allows aliasing path of 
   model storage path to session corresponding to routing, which represents 
   the service. default value is ``/models``.
-  Model Definition File Path: You can also set model definition file as you

>    uploaded in model storage path. default value is ``model-definition.yaml``.

Reviewer: ```suggestion
   uploaded in the model storage path. The default value is ``model-definition.yaml``.
```

   any AI accelerator on demand according to resource group.

.. image:: cluster_mode.png
   :width: 700
   :align: center

-  Single Node : When running a session, the managed node and worker nodes are 
   placed on a single physical node or virtual machine.
-  Multi Node : When running a session, one managed node and one or more worker 
   nodes are split across multiple physical nodes or virtual machines.

Before creating model service, Backend.AI supports validation feature to check 

> whether execution is available or not(due to any errors during execution.) By 
> clicking ``Validate`` button in bottom-left of service launcher, a new popup for 
> listening validation event will pops up. in the popup modal, you can check the status 
> by container log. when result is set to ``Finished``, then validation check is finished.

Reviewer: ```suggestion
whether execution is available or not(due to any errors during execution.) By clicking the ``Validate`` button at the bottom-left of the service launcher, a new popup for listening to validation events will pop up. In the popup modal, you can check the status through the container log. When the result is set to ``Finished``, then the validation check is finished.
```

   any AI accelerator on demand according to resource group.

.. image:: cluster_mode.png
   :width: 700
   :align: center

> -  Single Node:When running a session, the managed node and worker nodes are 

Reviewer: ```suggestion
-  Single Node: When running a session, the managed node and worker nodes are 
```
